WORKSPACE

```diff
@@ -13,15 +13,22 @@ http_archive(
     url = "https://github.com/bazelbuild/rules_go/releases/download/0.10.3/rules_go-0.10.3.tar.gz",
     sha256 = "feba3278c13cde8d67e341a837f69a029f698d7a27ddbb2a202be7a10b22142a",
 )
+http_archive(
+    name = "bazel_gazelle",
+    url = "https://github.com/bazelbuild/bazel-gazelle/releases/download/0.10.1/bazel-gazelle-0.10.1.tar.gz",
```

Comment:
this isn't the same version that we have vendored now. (we should probably not be vendoring gazelle twice.)

Reply:
(maybe worth fixing in a followup; if we commit to requiring bazel for test-infra, then we can remove the copy of gazelle in `vendor/`.)

Reply:
Done